modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -76,7 +88,7 @@ const renderInlineDialog = <T extends Dialog.DialogData>(dialogInit: DialogManag
   const dialog = GuiFactory.build({
     dom: {
       tag: 'div',
-      classes: [ 'tox-dialog', inlineClass ],
+      classes: [ 'tox-dialog', inlineClass ].concat(dialogSize),
```

Comment:
I would recommend to use spread operator over concat.

Reply:
Or even make getInlineDialogSizeClasses return `Optional<string>` and you can do a fold here.

Reply:
Using an Optional is probably overkill. Spread should be fine 👍